SPECS/nginx/nginx.spec

```diff
@@ -108,8 +128,8 @@ exit 0
 %config(noreplace) %{_sysconfdir}/%{name}/scgi_params.default
 %config(noreplace) %{_sysconfdir}/%{name}/uwsgi_params
 %config(noreplace) %{_sysconfdir}/%{name}/uwsgi_params.default
-%{_sysconfdir}/%{name}/win-utf
 %{_sysconfdir}/%{name}/html/*
+%{_sysconfdir}/%{name}/win-utf
```

Comment:
nit - unnecessary change

Reply:
Ahh this must have been the `sort` when i did it ..